pkgs/development/libraries/libpqxx/default.nix

```diff
@@ -1,24 +1,24 @@
-{ lib, stdenv, fetchFromGitHub, postgresql, doxygen, xmlto, python2, gnused }:
+{ lib, stdenv, fetchFromGitHub, postgresql, python39, gnused }:
 
 stdenv.mkDerivation rec {
   pname = "libpqxx";
-  version = "6.4.5";
+  version = "7.5.2";
 
   src = fetchFromGitHub {
     owner = "jtv";
     repo = pname;
     rev = version;
-    sha256 = "0djmjr2b5x5nd2a4idv5j8s6w0kdmvil910iv1kyc7x94dirbrni";
+    sha256 = "15ifd28v6xbbx931icydy8xmkd8030b20xzqjja6vwwvzss2w9fa";
   };
 
-  nativeBuildInputs = [ gnused python2 ];
-  buildInputs = [ postgresql doxygen xmlto ];
+  nativeBuildInputs = [ gnused python39 ];
```

Comment:
Can we use python3?

Reply:
`python3Full`? Should work it just wasn't clear which python 3 package I was supposed to use.

Reply:
he meant, just `python3`, so this gets updated when the default python3 gets updated (currently it's python38)

Reply:
oh I didn't know realize that existed It doesn't show up on search.nixos.org